wp-content/themes/access/views/partials/category-icon.twig

```diff
@@ -9,9 +9,7 @@
   {% else %}
     {% set catSlug = category.slug %}
   {% endif %}
-  <a href="{{ post.get_permalink }}" aria-label={{ catSlug }} {% if target %}target="{{ target }}"{% endif %}>
     <svg class="icon icon-card-{{ catSlug }}"{% if width %} width="{{ width }}"{% endif %}{% if height %} height="{{ height }}"{% endif %}>
```

Comment:
@theasteve I completely forgot that we need to have the category label in the SVG element. On the Icons documentation in the patterns there is a description on how to add the `role="img"` to the `<svg>` tag as well as the `<title>` tag within the SVG where the category label should be stored. The ID is also needed on the title tag for screen reader support. https://cityofnewyork.github.io/ACCESS-NYC-PATTERNS/icons

This should be the final code;

```
<svg class="icon icon-card-{{ catSlug }}"{% if width %} width="{{ width }}"{% endif %}{% if height %} height="{{ height }}"{% endif %} role="img">
  <title id="icon-card-{{ catSlug }}__title">{{ category.name }}</title>
  <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-card-{{ catSlug }}"></use>
</svg>
```
It should be good to go after that.